frontend/src/styles/base/_spacings.scss

```diff
@@ -0,0 +1,17 @@
+// Spacing scale
+$spacing-1: 0.5rem;
```

Comment:
Curious, where did you get these values?

Reply:
It's in the design system. The designers defined some fixed set of spacing for laying out elements. 

![image](https://user-images.githubusercontent.com/3666479/98057676-ca608300-1e7d-11eb-8b59-43582f56d286.png)

Reply:
Thanks for pointing this out, I didn't scroll to the right on figma.